priv/repo/migrations/20160610143806_add_meta_extra_index.exs

```diff
@@ -0,0 +1,14 @@
+defmodule HexWeb.Repo.Migrations.AddMetaExtraIndex do
+  use Ecto.Migration
+
+  def up do
+    execute """
+      CREATE INDEX packages_meta_extra ON
+        packages USING GIN ((meta->'extra'))
```

Comment:
We should use the index operator `jsonb_path_ops`, i.e. `USING GIN (meta->'extra' jsonb_path_ops)`. While you're at it append `_idx` to the index name :).

This is my final comment I believe, will merge after this change.